_index.md

```diff
@@ -0,0 +1,104 @@
+
+Each session has its own folder.
+The `index.md` file from a session folder details the topic of the session and includes practical demos of tools and snippets of code.
+
+After reading the session contents, jump to the hands-on activities.
+A typical activity will challenge you to solve / exploit a given issue and obtain a flag.
+Read the activity description in the session contents and then use the activity support file to get to a solution.
+Each activity will develop your practical skills and help you get a better understanding of the session topics.
+
+Activities come in two forms:
+
+- **tutorials**: that provide a step-by-step detailing of the solution.
+- **challenges**: that only state the goal and provide an initial scaffolding (and maybe some hints), then the rest is up to you.
+  Even so, challenge activity folders store the reference solution for you to take a sneak peak in case you're lost.
+
+## Support
+
+In case you have questions or want to know more about a topic, use the `Discussions` tab or join the [SSS Discord community](https://bit.ly/DiscordSecuritySummerSchool).
+
+## Sessions
+
+The sessions of the SSS **TODO-track-name** track are:
+
+- [Session 01: Name](session-name-1/index.md)
+- TODO
+- TODO
+
+### Session Folder Structure
+
+A session folder consists of:
+
+- `index.md`: session contents, written in [GitHub Markdown](https://guides.github.com/features/mastering-markdown/)
+- `activities/`: each activity has its own folder with support files (including the reference solution)
+- `assets/`: support files for session contents (images, diagrams, code snippets, demos, packet captures, etc.)
+- `slides/`: Markdown file used to generate the session slides.
+
+### Activity Folder Structure
+
+An activity folder consists of:
+
+- `README.md`: details the challenge name, challenge description, points, hints, vulnerability overview, solution overview.
+- `flag` (optional): contains the flag string.
+- `public/`: stores public files that are to be packed in an archive that is to be made available to participants.
+  They are usually generated from the `src/` folder, though they may also be static files.
+- `sol/`: the reference solution of the activity.
+- `src/` (optional): stores similar contents to a CTF challenge.
+  The generated files that are to be made available to participants will be stored in the `public/` folder in the corresponding activity folder in the public repository.
+  Most activities require a source code folder, but some may not, making the `src/` folder optional.
+- `deploy/` (optional): stores similar contents to a CTF challenge.
+  Most activities require remote deployment in a Docker container, but some may not, making the `deploy` folder optional.
+
+  It typically consists of the following files:
+
+  - `Dockerfile`: for creating the Docker container that will run the challenge on the remote system.
+  - `docker-compose.yml`: for configuring the Docker containers
```

Comment:
I would mention the `.dockerignore` file and reference [the documentation](https://docs.docker.com/engine/reference/builder/#dockerignore-file).

Reply:
Done.